src/ui/localization.ts

```diff
@@ -140,6 +140,138 @@ const localizeInit = () => {
                         "tooltip.local-space": "Local Space Gizmo"
                     }
                 },
+                fr: {
+                    translation: {
+                        // scene menu
+                        "scene": "Scène",
+                        "scene.new": "Créer",
+                        "scene.open": "Ouvrir",
+                        "scene.import": "Importer",
+                        "scene.load-all-data": "Charger toutes les données ply",
+                        "scene.save": "Enregistrer",
+                        "scene.save-as": "Enregistrer sous...",
+                        "scene.export": "Exporter",
+                        "scene.export.compressed-ply": "Ply compressé",
+                        "scene.export.splat": "Fichier splat",
+
+                        // selection menu
+                        "selection": "Sélection",
+                        "selection.all": "Tout",
+                        "selection.none": "Aucune",
+                        "selection.invert": "Inverser",
+                        "selection.lock": "Verrouiller la sélection",
+                        "selection.unlock": "Tout débloquer",
+                        "selection.delete": "Supprimer la sélection",
+                        "selection.reset": "Réinitialiser splat",
+
+                        // help menu
+                        "help": "Aide",
+                        "help.shortcuts": "Raccourcis claviers",
+                        "help.user-guide": "Guide utilisateur",
+                        "help.log-issue": "Signaler un problème",
+                        "help.github-repo": "Dépôt GitHub",
+                        "help.discord": "Serveur Discord",
+                        "help.forum": "Forum",
+                        "help.about": "À propos de supersplat",
+
+                        // modes
+                        "mode.centers": "Mode centres",
+                        "mode.rings": "Mode anneaux",
+
+                        // scene panel
+                        "scene-manager": "GESTIONNAIRE DE SCENE",
+                        "transform": "TRANSFORMATION",
+                        "position": "Position",
+                        "rotation": "Rotation",
+                        "scale": "Échelle",
+
+                        // options panel
+                        "options": "VOIR LES OPTIONS",
```

Comment:
I think this isn't quite right. This translated to "View the options" right? VIEW OPTIONS is options/settings related to the view (or the 3D viewport). For example, whether to show spherical harmonics or splat centers in the 3D view.